docs/Kube-API-examples.md

```diff
@@ -10,3 +10,33 @@ You will likely need to adapt those for your own needs.
 * [Hive PullSecret Secret](crds/pullsecret.yaml)
 * [Hive ClusterDeployment](crds/clusterDeployment.yaml)
 * [Hive ClusterDeployment-SNO](crds/clusterDeployment-SNO.yaml)
+
+
+
+###Creating InstallConfig overrides
+
+In order to alter the default install config yaml used when running `openshift-install create` commands.
+[REST-API reference](user/guide/install-customization.md)
```

Comment:
I don't see any guidance in those docs about how to compose the override value. This would be a good time to add a little blurb about what it should look like and what to expect.